### PR TITLE
Enable Hydra Configs in Stokes and GraphCast example, some doc additions to Stokes example

### PR DESCRIPTION
**Pull request by ktangsali**
# Modulus Pull Request

## Description
Add support for hydra configs.

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

File: examples/cfd/stokes_mgn/inference.py [hunk 1]
```diff
@@ -45,37 +48,38 @@
         + "pip install pyvista"
     )
 
-C = Constants()
-
 
 class MGNRollout:
-    def __init__(self, wb):
+    def __init__(self, cfg: DictConfig, wb, logger):
```
Comment (Alexey-Kamenev) [inline examples/cfd/stokes_mgn/inference.py:53]: Note on `wb`: this is the `wandb` module alias, not an instance, so makes little sense to pass it all over the place as an argument.
I would suggest replacing all occurrences of `wb` with just `wandb` (similar to what I did in Ahmed body and vortex shedding samples).

File: examples/cfd/stokes_mgn/train.py [hunk 2]
```diff
@@ -37,38 +41,35 @@
...
 from utils import relative_lp_error
-from constants import Constants
-
-# Instantiate constants
-C = Constants()
 
 
 class MGNTrainer:
-    def __init__(self, wb, dist, rank_zero_logger):
+    def __init__(self, cfg: DictConfig, wb, dist, rank_zero_logger):
```
Comment (Alexey-Kamenev) [inline examples/cfd/stokes_mgn/train.py:47]: Same comment about `wb` as before.

File: examples/cfd/stokes_mgn/train.py [hunk 3]
```diff
@@ -195,37 +198,30 @@ def validation(self):
...
         entity="Modulus",
         name="Stokes-Training",
         group="Stokes-DDP-Group",
-        mode=C.wandb_mode,
+        mode=cfg.wandb_mode,
     )
 
     logger = PythonLogger("main")  # General python logger
     rank_zero_logger = RankZeroLoggingWrapper(logger, dist)  # Rank 0 logger
     logger.file_logging()
```
Comment (Alexey-Kamenev) [inline examples/cfd/stokes_mgn/train.py:218]: `logger` -> `rank_zero_logger`?

File: examples/weather/graphcast/conf/config.yaml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+save_freq: 1  # 500
+val_freq: 1  # 1000
+ckpt_path: "checkpoints_34var"
+val_dir: "validation_34var"
+ckpt_name: "model_34var.pt"
+use_apex: False
+pyt_profiler: False
+profile: False
+profile_range: (90, 110)
+icospheres_path: "icospheres.json"
```
Comment (Alexey-Kamenev) [inline examples/weather/graphcast/conf/config.yaml:70]: Nit: insert final new line.

**Review by Alexey-Kamenev — APPROVED** [review]
LGTM with minor comments.

File: examples/cfd/stokes_mgn/inference.py (hunk 1 above)
Comment (ktangsali) [reply examples/cfd/stokes_mgn/inference.py]: Thanks, resolved!

**Review by mnabian — APPROVED** [review]

**Pull request merged**